DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/dcb/linux/linux_dcb_app_impl.py

```diff
@@ -0,0 +1,62 @@
+import json
+from dent_os_testbed.lib.dcb.linux.linux_dcb_app import LinuxDcbApp
+
+
+class LinuxDcbAppImpl(LinuxDcbApp):
+    """
+        dcb [ OPTIONS ] { app | buffer | ets | maxrate | pfc } { COMMAND | help }
+        dcb [ -force ] -batch filename
+        dcb [ OPTIONS ] help
+
+    """
+    def format_set(self, command, *argv, **kwarg):
+        """
+        dcb app {  add  |  del  |  replace  }  dev DEV [ default-prio
+                PRIO-LIST ] [ ethtype-prio ET-MAP ] [ stream-port-prio
+                PORT-MAP ] [ dgram-port-prio PORT-MAP ] [ port-prio PORT-
+                MAP ] [ dscp-prio DSCP-MAP ]
+
+        """
+        params = kwarg["params"]
+        cmd = "dcb {} app {} dev {} ".format(
+            params.get("options", ""),
+            command,
+            params["dev"])
+
+        if "default_prio" in params:
+            cmd += "default-prio " + " ".join(map(str, params["default_prio"]))
+        if "dscp_prio" in params:
+            dscp_prio_map = (f"{dscp}:{prio}" for dscp, prio in params["dscp_prio"])
+            cmd += "dscp-prio " + " ".join(dscp_prio_map)
```

Comment:
Correct me if I'm wrong, but as I recall, ethtype-prio, stream-port-prio, dgram-port-prio and port-prio should have the same command syntax as for dscp-prio. It means that we can add here a universal approach for each of them which will cover command fully.